www/__mocks__/fileSystemMocks.ts

```diff
@@ -0,0 +1,18 @@
+export const mockFileSystem = () => {
+    window['resolveLocalFileSystemURL'] = function (parentDir, handleFS) {
+        const fs = {"filesystem": 
```

Comment:
It doesn't matter a ton but just FYI, JS object declarations don't need quotes around property names (`filesystem:`) and it's somewhat preferred to not have them

Reply:
Good to know! I went ahead and resolved that since it was an easy fix.